app/public/model/logged/modelData/FetchedDataManager.js

```diff
@@ -77,8 +77,7 @@ export default class FetchedDataManager {
 
         let totalRecordsNumber = null;
         if (!countAllRecord) {
-            // eslint-disable-next-line prefer-destructuring
-            totalRecordsNumber = this[page][index].payload.totalRecordsNumber;
+            ({ totalRecordsNumber } = this[page][index].payload);
```

Comment:
@xsalonx Można to napisać trochę bardziej fancy, przy okazji eslint nie będzie krzyczał :smirk:
I ogólnie dość fajna sprawa:
![eslintt](https://user-images.githubusercontent.com/48785655/174395016-651735d2-08d6-4e56-9736-be468e73fb38.png)